src/renderer/rendered_window.rs

```diff
@@ -372,42 +372,6 @@ impl RenderedWindow {
                 renderer.draw_foreground(canvas, &cells, grid_position, width, &style);
                 canvas.restore();
             }
-            WindowDrawCommand::Scroll {
-                top,
-                bot,
-                left,
-                right,
-                rows,
-                cols,
-            } => {
-                let scrolled_region = Rect::new(
-                    (left * renderer.font_width) as f32 * scaling,
-                    (top * renderer.font_height) as f32 * scaling,
-                    (right * renderer.font_width) as f32 * scaling,
-                    (bot * renderer.font_height) as f32 * scaling,
```

Comment:
Problem here is definitely with this scaling.

And i think we can somehow fix it, without removing this optimization.

Here is example with scale factor 1.6
![image](https://user-images.githubusercontent.com/301015/124428929-e9e5dd80-dd75-11eb-9897-5377526b9bc7.png)

Here is example with scale factor 2:
![image](https://user-images.githubusercontent.com/301015/124429101-1e599980-dd76-11eb-9abd-ebbdf483036c.png)

No artifacts!

But i noticed something else interesting, when using `--multiGrid`, there is no artifacts, with either scroll factor.